uefi/src/lib.rs

```diff
@@ -74,8 +77,48 @@
 //! only unfold their potential when you invoke `uefi::helpers::init` as soon
 //! as possible in your application.
 //!
+//! # Discuss and Contribute
+//!
+//! For general discussions, feel free to join us in our [Zulip] and ask
+//! your questions there.
+//!
+//! Further, you can submit bugs and also ask questions in our [issue tracker].
+//! Contributions in form of a PR are also highly welcome. Check our
+//! [contributing guide][contributing] for details.
+//!
+//! # MSRV
+//! <!-- Keep in Sync with README! -->
+//!
+//! The minimum supported Rust version is currently 1.70.
+//! Our policy is to support at least the past two stable releases.
+//!
+//! # License
+//! <!-- Keep in Sync with README! -->
+//!
+//! The code in this repository is licensed under the Mozilla Public License 2.
+//! This license allows you to use the crate in proprietary programs, but any
+//! modifications to the files must be open-sourced.
+//!
+//! The full text of the license is available in the [license file][LICENSE].
+//!
+//! # Trivia and Background
```

Comment:
I think we should drop this section in the interest of keeping the crate docs focused on the most useful information. The history of UEFI is interesting, but not particularly relevant to developing an app with uefi-rs.

Reply:
I just drop the entire paragraph. It is irrelevant. We are already referring to the UEFI spec. This is sufficient